hello-world-modularized/lib/hello.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function() {
+  console.log('hey world!');
+};
```

Comment:
This file isn't necessary for the assignment, just an organization nitpick :) We can go over what should be expected for these submissions if you would like more clarification!